src/forms/style.scss

```diff
@@ -60,6 +60,27 @@ $input-default-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
   }
 }
 
+.final-form-select.is-disabled {
+  .Select-control {
```

Comment:
just wondering, why do we need special css? isnt it by default by bootstrap/patternfly when the disable attribute is set?

Reply:
apparently no. Adding just the disabled prop has no effect on the component.